### PR TITLE
adding docs to clnapp

### LINKED ISSUE #3: docs: add a well formatted readme

### PR DESCRIPTION
**Pull request by KAVAN-DESAI**
Fixes #3

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,19 +1,104 @@
-# clnapp
+<div align="center">
+  <h1>lndart.clnapp</h1>
+
+  <img src="https://github.com/dart-lightning/icons/raw/main/main/res/mipmap-xxxhdpi/ic_launcher.png" />
+
+  <p>
+    <strong> :dart: An app to manage core lightning node using multiple clients :dart: </strong>
```
Comment (vincenzopalazzo) [inline README.md:7]:
```suggestion
    <strong> :dart: Cross platform app to manage remotely and locally a core lightning  :dart: </strong>
```

File: README.md [hunk 2]
```diff
@@ -1,19 +1,104 @@
...
+- Clone the lndart.clnapp repository.
+
+```bash
+sudo git clone https://github.com/dart-lightning/lndart.clnapp.git
+```
+- Get flutter dependencies
+
+```bash
+flutter pub upgrade
+flutter pub get
```
Comment (vincenzopalazzo) [inline README.md:66]:
```suggestion
make dep
make
```

File: docs/dev/MAINTAINERS.md [hunk 3]
```diff
@@ -90,8 +89,9 @@ The version in our package.json gets copied to the one we publish, and users nee
 
 ### Scopes
 
-- **clnapp**: Changes related to the cln client
-- **clnapp_flutter**: Changes related to the clnapp_flutter package
+- **cln_client**: Changes related to the cln client
+- **clnapp**: Changes related to the clnapp view or for some app specific changes
+- **clnapp_flutter**: Changes related to the clnapp flutter package
```
Comment (vincenzopalazzo) [inline docs/dev/MAINTAINERS.md:94]:
```suggestion
- **clnapp**: Changes related to the clnapp view or for some app specific changes
```

we have only an app here with a single package

**Review by vincenzopalazzo — CHANGES_REQUESTED** [review]
Change you add also a `changelog.json`?

File: changelog.json [hunk 4]
```diff
@@ -0,0 +1,16 @@
...
+  "package_name": "clnapp",
+  "version": "v0.0.1-alpha.2",
+  "api": {
+    "name": "github",
+    "repository": "dart-lightning/lndart.clnapp",
+    "branch": "main"
+  },
+  "generation_method": {
+    "name": "semver-v2",
+    "header_filter": true
```
Comment (vincenzopalazzo) [inline changelog.json:11]:
```suggestion
    "header_filter": false
```

This is used when you have somethings like multipackage, so you can filter by scope es. `fix(plugin)`

Comment (KAVAN-DESAI) [reply changelog.json:11]: ooh okay thanks

**Pull request merged**